taxcalc/records.py

```diff
@@ -684,3 +860,64 @@ def imputation(e17500, e00100, e18400, e18425, e62100, e00700, e04470,
                        - state_adjustment - e00100 - e18500 - e20800)
 
     return _cmbtp_itemizer
+
+
+def read_blowup(blowup_factors):
+    if isinstance(blowup_factors, pd.core.frame.DataFrame):
+        BF = blowup_factors
+    else:
+        try:
+            if not os.path.exists(blowup_factors):
+                # grab blowup factors out of EGG distribution
+                path_in_egg = os.path.join("taxcalc",
+                                           BLOWUP_FACTORS_FILENAME)
+                blowup_factors = resource_stream(Requirement.parse("taxcalc"),
+                                                 path_in_egg)
+
+            BF = pd.read_csv(blowup_factors, index_col='YEAR')
+        except IOError:
+            print("Missing a csv file with blowup factors. Please pass "
+                  "such a csv as PUF(blowup_factors='[FILENAME]').")
+            raise
+            # TODO, we will need to pass the csv to the Calculator once
```

Comment:
this comment is old and should have been removed a while ago. Best to get rid of it now.